examples/face_landmarker/ios/FaceLandmarkerUITests/FaceLandmarkerUITestsLaunchTests.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  FaceLandmarkerUITestsLaunchTests.swift
+//  FaceLandmarkerUITests
+//
+//  Created by MBA0077 on 6/23/23.
```

Comment:
Who?

Does this test do anything of value? We should either add to it to make sure it does, or remove it.